### DISCUSSION TITLE
Compatibilità PHP

### DISCUSSION THREAD

**Discussion by albertovincenzi** [masterix21/green-pass#5] [Q&A]
Ciao,
non ho controllato tutto il sorgente, ma sei sicuro che php 8.0 sia richiesto? Seppur ottimo, non sarebbe meglio rendere la libreria compatibile ancora con la versione 7?

**Comment by masterix21** [comment]
La validazione fa uso della funzione `str_starts_with` di PHP8. Non utilizzo PHP7 quindi non concentro mai i miei sforzi su cose che non uso.

Se pensi che possa essere utile, accetto un PR per aggiungere il supporto a PHP7.
